lib/DxilContainer/DxilContainerAssembler.cpp

```diff
@@ -1753,9 +1754,7 @@ void hlsl::SerializeDxilContainerForModule(
     DXASSERT(pModule->GetSerializedRootSignature().empty(),
              "otherwise, library has root signature outside subobject definitions");
     // Write the DxilCompilerVersion (VERS) part.        
-    unsigned ValMajor, ValMinor;
-    pModule->GetValidatorVersion(ValMajor, ValMinor);
-    bool bValidatorAtLeast_1_8 = DXIL::CompareVersions(ValMajor, ValMinor, 1, 8) >= 0;
+
```

Comment:
nit: there's some trailing whitespace on the prior line and an extra newline here.